lib/brisket.js

```diff
@@ -37,6 +37,8 @@ var Brisket = {
 
     $: require("./application/jquery"),
 
+    version: require("../package.json").version,
```

Comment:
does this mean that apps will have to bundle the package json into user's browsers? that could be considered a security risk for some apps. is there precedent of importing package.json in other user facing projects?